test-apps/display-test-app/package.json

```diff
@@ -63,9 +64,12 @@
     "@itwin/editor-common": "workspace:*",
     "@itwin/appui-abstract": "workspace:*",
     "@itwin/webgl-compatibility": "workspace:*",
+    "@itwin/oidc-signin-tool": "^3.2.2",
```

Comment:
Why is the oidc signin tool needed? This performs a headless automated sign-in which should only be needed when doing integration testing.

Reply:
This is for `TestBrowserAuthorizationClient`/`TestFrontendAuthorizationClient` for use with totally automated operation. I copied the code that made use of that from the recently update diplay-performance-test-app [PR](https://github.com/iTwin/itwinjs-core/pull/3822/files).